app/models/calculation_result.rb

```diff
@@ -62,143 +66,205 @@ def assessed_capital
+      employment_expenses: employment_deduction(:fixed_employment_deduction, prefix),
+    }
+    dependants_allowance = api_response.dig(:result_summary, :"#{prefix}disposable_income", :dependant_allowance)
+    partner_allowance = api_response.dig(:result_summary, :"#{prefix}disposable_income", :partner_allowance)
+
+    data[:dependants_allowance] = dependants_allowance if dependants_allowance&.positive?
+    data[:partner_allowance] = partner_allowance if partner_allowance&.positive?
+    data.transform_values { |v| monetise(v) }
+  end
+
+  def capital_rows(prefix: nil)
+    data = {
+      liquid: api_response.dig(:result_summary, :"#{prefix}capital", :total_liquid),
+      non_liquid: api_response.dig(:result_summary, :"#{prefix}capital", :total_non_liquid),
+      property: api_response.dig(:result_summary, :"#{prefix}capital", :total_property),
+      vehicles: api_response.dig(:result_summary, :"#{prefix}capital", :total_vehicle),
+    }
+    data.transform_values { |value| monetise(value) }
+  end
+
+  def capital_subtotal_rows(prefix: nil)
+    data = {
+      total_capital: api_response.dig(:result_summary, :"#{prefix}capital", :total_capital),
+      pensioner_capital_disregard: api_response.dig(:result_summary, :"#{prefix}capital", :pensioner_capital_disregard),
+      smod_disregard: api_response.dig(:result_summary, :"#{prefix}capital", :subject_matter_of_dispute_disregard),
+    }
+
+    data.transform_values { |value| monetise(value) }
+  end
+
+  def main_home_rows(prefix: nil)
+    data = {
+      main_home_value: :value,
+      main_home_mortgage: :outstanding_mortgage,
+      main_home_disregard: :main_home_equity_disregard,
+      main_home_equity: :assessed_equity,
+    }
+    data.transform_values { |v| monetise(capital_items(:properties, prefix).dig(:main_home, v)) }
+  end
+
+  def additional_property_rows(prefix: nil)
+    data = {
+      additional_property_value: :value,
+      additional_property_mortgage: :outstanding_mortgage,
+      additional_property_equity: :assessed_equity,
+    }
+    data.transform_values { |v| monetise(capital_items(:properties, prefix)[:additional_properties].first[v]) }
+  end
+
+  def vehicle_rows(prefix: nil)
+    data = {
+      vehicle_value: capital_items(:vehicles, prefix).sum(0) { |z| z.fetch(:value) },
+      vehicle_outstanding_payments: capital_items(:vehicles, prefix).sum(0) { |z| z.fetch(:loan_amount_outstanding) },
+      vehicle_disregards: vehicle_disregards(prefix),
+      vehicle_assessed_value: capital_items(:vehicles).sum(0) { |z| z.fetch(:assessed_value) },
+    }
+
+    data.transform_values { |value| monetise(value) }
+  end
+
+  def vehicle_disregards(prefix)
+    value = capital_items(:vehicles, prefix).sum(0) { |z| z.fetch(:value) }
+    pcp = capital_items(:vehicles, prefix).sum(0) { |z| z.fetch(:loan_amount_outstanding) }
+    assessed_value = capital_items(:vehicles, prefix).sum(0) { |z| z.fetch(:assessed_value) }
+    value - pcp - assessed_value
```

Comment:
should we be doing this calculation here, or should CFE be doing it?

Reply:
You're right, CFE should, and now that we're being more hands-on with CFE, it's within our ability to make that happen. I'll get on it.